x-pack/platform/plugins/shared/alerting/server/application/rule/methods/bulk_edit_params/schemas/bulk_edit_rule_params_option_schemas.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema } from '@kbn/config-schema';
+
+const exceptionListSchema = schema.object({
+  id: schema.string(),
+  list_id: schema.string(),
+  type: schema.oneOf([
```

Comment:
It looks like [our copy](https://github.com/rylnd/kibana/blob/validate-rule-params-edit/x-pack/solutions/security/plugins/security_solution/common/api/detection_engine/model/rule_schema/common_attributes.gen.ts#L666) of the list types is not compatible with the one defined here, as ours has an extra key of `endpoint_trusted_devices` (which was added two weeks ago).

Can you reuse some of these types, or does it make sense to have separate definitions in each area?